### PR TITLE
VIA Support: KBD67 mkii v1/v2

### PR DESCRIPTION
**Pull request by mechmerlin**
## Description

Add the VIA keymap and change vendor id to something thats not hopefully been used yet. I picked "KB". 

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/kbdfans/kbd67/mkiirgb/v1/config.h [hunk 1]
```diff
@@ -1,10 +1,10 @@
 #pragma once
 #include "config_common.h"
-#define VENDOR_ID       0xFEED
+#define VENDOR_ID       0x4b42 // KB
```
Comment (noroadsleft) [inline keyboards/kbdfans/kbd67/mkiirgb/v1/config.h]: The hex value should be uppercase.

```suggestion
#define VENDOR_ID       0x4B42 // KB
```

File: keyboards/kbdfans/kbd67/mkiirgb/v2/config.h [hunk 2]
```diff
@@ -1,10 +1,10 @@
 #pragma once
 #include "config_common.h"
-#define VENDOR_ID       0xFEED
-#define PRODUCT_ID      0x1224
+#define VENDOR_ID       0x4b42 // KB
```
Comment (noroadsleft) [inline keyboards/kbdfans/kbd67/mkiirgb/v2/config.h]:
```suggestion
#define VENDOR_ID       0x4B42 // KB
```

**Comment by mechmerlin** [comment]
ooof something got mangled in my 65_ansi_blocker community layout

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**